src/sentry/templates/sentry/integrations/slack-reauth-details.html

```diff
@@ -57,8 +57,8 @@ <h3>{% trans "Time For An Upgrade" %}</h3>
             {% endfor %}
             <p class="section-info">
                 {% blocktrans %}
-                We can't even remember the name of that show everyone's been recommending, so we couldn't possibly
-                expect you to remember these channel names. We'll send you another reminder in your private Slack channels.
+                No need to waste time committing these to memory — after you authenticate the app,
```

Comment:
@MeredithAnya I think this should be two sentences. Personally, I hate using a "-" like this.

Reply:
This was from molly's other options for copy (I personally love "-" :)) but I don't have a preference so I can update